projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { Text } from '@automattic/jetpack-components';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import ProductCard from '../../connected-product-card';
+import useVideoPressCardDescription from './use-videopress-description';
+import VideoPressValueSection from './videopress-value-section';
+import type { ProductCardType } from '../types';
+
+import './style.scss';
+
+const VideopressCard: ProductCardType = ( { admin } ) => {
+	const slug = 'videopress';
+	const { detail } = useProduct( slug );
+	const { isPluginActive = false } = detail || {};
+	const { videopress: data } = getMyJetpackWindowInitialState();
+
+	const descriptionText = useVideoPressCardDescription( {
+		isPluginActive,
+		videoCount: data.videoCount,
+	} );
+
+	const Description = () => (
+		<Text variant="body-small" className="description">
+			{ descriptionText }
+		</Text>
+	);
+
+	return (
+		<ProductCard
+			slug={ slug }
+			showMenu
+			admin={ admin }
+			Description={ descriptionText ? Description : null }
```

Comment:
Maybe `Description` variable could be already `null` if `descriptionText` is not provided?

Reply:
I tried this but Description is a function so it does not return null until after it is passed to the function. I think we need to do it inline here 🤔

Reply:
Yeah, my bad! I thought we can do something like this ` useCallback( descriptionText ? () => <Description /> : null, [ descriptionText ] );` But I see we cannot.